src/pages/workspace/accounting/qbo/export/QuickbooksExportConfigurationPage.tsx

```diff
@@ -84,46 +72,43 @@ function QuickbooksExportConfigurationPage({policy}: WithPolicyConnectionsProps)
     ];
 
     return (
-        <AccessOrNotFoundWrapper
-            policyID={policyID}
+        <ConnectionLayout
+            displayName={QuickbooksExportConfigurationPage.displayName}
+            headerTitle="workspace.accounting.export"
+            title="workspace.qbo.exportDescription"
             accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
+            policyID={policyID}
             featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            contentContainerStyle={styles.pb2}
+            titleStyle={styles.ph5}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.QBO}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING.getRoute(policyID))}
```

Comment:
Looks like having a fallback to the `goBack` function here led to the following issue:
- https://github.com/Expensify/App/issues/48630

which was fixed by removing the fallback, more details in the [proposed solution](https://github.com/Expensify/App/issues/51107#issuecomment-2423370557).